clients/java/src/main/java/io/camunda/zeebe/client/impl/worker/JobWorkerBuilderImpl.java

```diff
@@ -73,7 +74,7 @@ public JobWorkerBuilderImpl(
     pollInterval = configuration.getDefaultJobPollInterval();
     requestTimeout = configuration.getDefaultRequestTimeout();
     enableStreaming = configuration.getDefaultJobWorkerStreamEnabled();
-    tenantIds = configuration.getDefaultJobWorkerTenantIds();
+    tenantIds = new ArrayList<>(configuration.getDefaultJobWorkerTenantIds());
```

Comment:
🔧 IMO, calling the `tenantId()` and `tenantIds()` methods should stack but also overwrite the default. It looks to me (but I could be wrong), that it currently doesn't overwrite the default. Could you add a test that shows that the default can be overwritten by calling the tenantId methods?

Reply:
it can't as of now, the test only works as there the `ZeebeClientConfiguration` is mocked and thus no default tenant is in the initial value 😅

Reply:
@megglos Perhaps you write the test without mocks 👍